tests/unit/customizations/test_paginate.py

```diff
@@ -54,6 +54,8 @@ def test_customize_arg_table(self):
             'foo': mock.Mock(),
             'bar': mock.Mock(),
         }
+        # Default to false
+        argument_table['foo']._UNDOCUMENTED = False
```

Comment:
Make sense. It was just strange that bar did not have its ``_UNDOCUMENTED`` set to ``False`` as well.